### PR TITLE
import clinvar updates for - e101

### PR DESCRIPTION
**Pull request by ima23**
- add evidence attribs to variation and variation feature
- add variation set ids to variation feature 
- add phenotype evidence const to Sequence utils
- update VariationAdaptor update method to avoid undef errors   
- created jira ticket for VariationFeatureAdaptor to include variation sets in the future ENSVAR-2717

### CODE REVIEW

File: scripts/import/import_clinvar_xml [hunk 1]
```diff
@@ -988,6 +1050,7 @@ sub enter_var{
       $ref_allele = $refSlice->seq;
       $allele_str = $ref_allele ."/". $alt_allele;
   }
+  print "Saving allele string: $vf->allele_string()\n";
```
[the remark this answers is not in the document]
Comment (ima23) [reply scripts/import/import_clinvar_xml]: No, it was a stashed lined from previous work. I removed it now.

File: scripts/import/import_clinvar_xml [hunk 2]
```diff
@@ -274,7 +286,7 @@ sub get_set{
 
   if (defined $set_id->[0]->[0] ){
     ## remove old set content
-    $set_del_sth->execute($set_id->[0]->[0]) if defined $clean;
+    $set_del_sth->execute($set_id->[0]->[0]) if defined $clean && $set eq 'ph_omim';
```
Comment (sarahhunt) [inline scripts/import/import_clinvar_xml]: It's confusing to only delete the OMIM set here. Is it safe to continue to delete all the ClinVar sets here? If not, a comment would be useful.

Comment (ima23) [reply scripts/import/import_clinvar_xml]: Thanks for checking @sarahhunt. Have removed it in the next commit.

File: scripts/import/import_clinvar_xml [hunk 3]
```diff
@@ -766,6 +806,28 @@ sub update_variation_set{
...
+      UPDATE variation_feature
+      SET
+        variation_set_id = CONCAT_WS(",", variation_set_id, ?),
+        evidence_attribs = CONCAT_WS(",", evidence_attribs, '$pheno_evidence_id')
+      WHERE variation_feature_id = ? ] );
+  my $new_sets = "$pheno_set_id,$clinvar_set_id";
+  if ($omimVarSet{$vf->variation->dbID}) {
+    $new_sets .=",$omim_set_id";
+  }
+  $vf_upd_sth->execute( $new_sets,  $vf->dbID );
```
Comment (sarahhunt) [inline scripts/import/import_clinvar_xml]: Which db server is this working on? I tried it on prod4 and got:
 DBD::mysql::st execute failed: Data truncated for column 'variation_set_id' at row 1 at scripts/import/import_clinvar_xml line 826.

It looks like '23,30,24' is being treated as a string.

Comment (ima23) [reply scripts/import/import_clinvar_xml]: Thanks for checking. There is a setup difference between prod4 and the other servers which leads to this error and not on the others, I created a jira ticked for it ENSVAR-2979.

**Pull request merged**